core/types/block.go

```diff
@@ -242,7 +242,8 @@ type extblock struct {
 	Header      *Header
 	Txs         []*Transaction
 	Uncles      []*Header
-	Withdrawals []*Withdrawal `rlp:"optional"`
+	Withdrawals []*Withdrawal  `rlp:"optional"`
```

Comment:
We want to keep the block body struct, the blobs field in the block is only for transferring blobs as extra data with the block. If we change the block body, maybe we have some compatibility issues, like block storage? block transfer?

We should discuss more about the change.